src/webgl/interaction.js

```diff
@@ -93,14 +93,18 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY, sensitivityZ) {
 
   const zoomScaleFactor = 0.02;
   const scaleFactor = this.height < this.width ? this.height : this.width;
+  const uP = this._renderer.uPMatrix.mat4;
 
   // ZOOM if there is a change in mouseWheelDelta
   if (this._mouseWheelDeltaY !== 0) {
     // zoom according to direction of mouseWheelDeltaY rather than value
-    if (this._mouseWheelDeltaY > 0) {
-      this._renderer._curCamera._orbit(0, 0, sensitivityZ * zoomScaleFactor);
-    } else {
-      this._renderer._curCamera._orbit(0, 0, - sensitivityZ * zoomScaleFactor);
+    const mouseWheelSign = (this._mouseWheelDeltaY > 0 ? 1 : -1);
+    const deltaRadius = mouseWheelSign * sensitivityZ * zoomScaleFactor;
+    this._renderer._curCamera._orbit(0, 0, deltaRadius);
+    // if ortho, scale change.
+    if (uP[15] !== 0) {
```

Comment:
Nice find, I didn't realize that `ortho` was the only one that has a 1 in that corner!

Reply:
I had to think about that. It would be nice if it could be determined by a flag or something else that it is an orthogonal projection, but "cameraType" cannot do so (ortho and frustum are the same, "custom"), so I decided to rely on the components.